.gitignore

```diff
@@ -1,3 +1,62 @@
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# Crash log files
+crash.log
+crash.*.log
+
+# Exclude all .tfvars files, which are likely to contain sensitive data, such as
```

Comment:
That is what `*.tfvars` is for

Reply:
So is the above an example of a `variables.tf` file being used incorrectly? I general I agree, I only use it for definitions. But we should add it to the gitignore.

Reply:
No, this is correct and should be checked in: https://github.com/hashicorp/terraform-elasticache-example/blob/master/variables.tf
This is incorrect and should not be checked in: https://github.com/hashicorp/terraform-elasticache-example/blob/master/terraform.tfvars

You can actually see why in the example, because the default values are ones that can be overriden, but the values in the `*.tfvars` file are specific to the user.

Reply:
Ok gotcha, so `variables.tf` can/may be checked in because they do not contain sensitive data. Makes sense, thanks for clarifying.